src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java

```diff
@@ -257,6 +257,50 @@ void shouldGetDatabaseMinorVersion() throws SQLException {
 		assertEquals(0, fireboltDatabaseMetadata.getDatabaseMinorVersion());
 	}
 
+	@Test
+	void getStringFunctions() throws SQLException {
+		getFunctions(DatabaseMetaData::getStringFunctions);
+	}
+
+	@Test
+	void getNumericFunctions() throws SQLException {
+		getFunctions(DatabaseMetaData::getNumericFunctions);
+	}
+
+	@Test
+	void getSystemFunctions() throws SQLException {
+		getFunctions(DatabaseMetaData::getSystemFunctions);
+	}
+
+	@Test
+	void getTimeDateFunctions() throws SQLException {
+		getFunctions(DatabaseMetaData::getTimeDateFunctions);
+	}
+
+	@ParameterizedTest
+	@CsvSource(value = {",true", "'',true", "abs,true", "SIN,true", "ThisFunctionDoesNotExist,false"}, delimiter = ',')
+	void getFunctions(String functionNamePattern, boolean filled) throws SQLException {
+		String previousFunction = null;
+		int count = 0;
+		for (ResultSet rs = fireboltDatabaseMetadata.getFunctions(null, null, functionNamePattern); rs.next();) {
+			count++;
+			String functionName = rs.getString("FUNCTION_NAME");
+			System.out.println(functionName);
```

Comment:
We probably don't want to print each time we run this test.

Reply:
definitely.